spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/onnx/Conv.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.intel.analytics.bigdl.nn.onnx
+
+import scala.reflect.ClassTag
+import com.intel.analytics.bigdl.nn
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+
+
+/**
+ * The convolution operator consumes an input tensor, and computes the output.
+ *
+ * auto_pad : string (default is NOTSET)
+ *  auto_pad must be either NOTSET, SAME_UPPER, SAME_LOWER or VALID.
+ *  Where default value is NOTSET, which means explicit padding is used.
+ *  SAME_UPPER or SAME_LOWER mean pad the input so that the output spatial size match the input.
+ *  In case of odd number add extra padding at the end for SAME_UPPER and the start for SAME_LOWER.
+ *  VALID mean no padding.
+ * dilations : list of ints. Dilation value along each spatial axis of the filter.
+ * group : int (default is 1), number of groups input channels and output channels are divided into.
+ * kernel_shape : list of ints.The shape of the convolution kernel.
+ * pads: Padding for the beginning and ending along each spatial axis,
+ * strides: Stride along each spatial axis.
+ */
+object Conv {
+  def apply[T: ClassTag](
+    nInputPlane: Int, // BigDL requires
+    nOutputPlane: Int, // BigDL requires
+    kernelShape: List[Int],
+    weight: Tensor[T], // BigDL requires
+    bias: Tensor[T], // BigDL requires
+    autoPad: String = "NOTSET", // missing in BigDL
```

Comment:
what's the meaning?

Reply:
The way how it should be padded automatically, it could be "Same, Valid"